### PR TITLE
correct link to pcre

### PR DESCRIPTION
**Pull request by qiaohaijun**
correct link is http://ftp.exim.llorien.org/pcre/pcre-8.39.tar.gz

### CODE REVIEW

**Comment by caisq** [comment]
@qiaohaijun can you please explain the reason behind this change? The old link seems to work fine, at east on my side.

File: tensorflow/workspace.bzl [hunk 1]
```diff
@@ -129,7 +129,7 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
   native.new_http_archive(
     name = "pcre",
     sha256 = "ccdf7e788769838f8285b3ee672ed573358202305ee361cfec7a4a4fb005bbc7",
-    url = "http://ftp.exim.org/pub/pcre/pcre-8.39.tar.gz",
+    url = "http://ftp.exim.llorien.org/pcre/pcre-8.39.tar.gz",
```
Comment (vrv) [inline tensorflow/workspace.bzl]: If exim.org is not available in your country, what about http://ftp.cs.stanford.edu/pub/exim/pcre/ instead?

**Comment by vrv** [comment]
(Closing, since you have to sign the CLA and our clabot seems to have not checked this)

File: tensorflow/workspace.bzl (hunk 1 above)
Comment (qiaohaijun) [inline tensorflow/workspace.bzl]: http://ftp.cs.stanford.edu/pub/exim/pcre/ is OK

**Pull request closed without merging**